packages/integrations/markdoc/README.md

```diff
@@ -249,6 +212,83 @@ export default defineMarkdocConfig({
 })
 ```
 
+### Set the root HTML element
+
+Markdoc wraps documents with an `article` tag by default. This can be changed from the `document` Markdoc node. This accepts an HTML element name or `null` if you prefer to remove the wrapper element:
+
+```js
+// markdoc.config.mjs
+import { defineMarkdocConfig, nodes } from '@astrojs/markdoc/config';
+
+export default defineMarkdocConfig({
+  nodes: {
+    document: {
+      render: null,
+      // Apply Markdoc's defaults for other options
+      ...nodes.document,
+    },
+  },
+})
+```
+
+### Custom Markdoc nodes / elements
+
+You may want to render standard Markdown elements like paragraphs and bolded text with components. For this, you can configure a [Markdoc node][markdoc-nodes]. If a given node receives attributes, they will be available as component props.
```

Comment:
```suggestion
You may want to render standard Markdown elements, such as paragraphs and bolded text, as Astro components. For this, you can configure a [Markdoc node][markdoc-nodes]. If a given node receives attributes, they will be available as component props.
```
I totally had to read this three times to not see "render standard Markdown elements as paragraphs..." :sweat_smile: I think this is what you mean?